corehq/apps/domain/urls.py

```diff
@@ -49,7 +49,7 @@
 def exception_safe_password_reset(request, *args, **kwargs):
     try:
         return password_reset(request, *args, **kwargs)
-    except None: 
+    except None:
```

Comment:
I realize you didn't really change this, but it will never catch an exception. Is that intended?